_translations/hu.yml

```diff
@@ -176,6 +176,7 @@ hu:
     walletelectrum: "Az Electrum fókuszában a sebesség és az egyszerűség áll, alacsony erőforrás-használattal párosulva. Távoli szervereket használ a Bitcoin rendszere legbonyolultabb részeinek kezeléshez, és lehetővé teszi pénztárcája visszaállítását egy titkos szó(kapcsolat) segítségével."
     walletmsigna: "Az mSIGNA egy haladó szintű, mégis egyszerűen használható pénztárca. amely gyors, letisztult, cégszintű használatra is alkalmas, valamint nagyon biztonságos. Támogatja a BIP32-t, a csoportos aláírású tranzakciókat, a többkészülékes szinkronizációt, valamint a titkosított elektronikus és papíralapú biztonsági másolatokat."
     walletbitcoinwallet: "A Bitcoin Wallet egyszerűen használható és megbízható, mindemellett pedig biztonságos és gyors. A fejlesztők víziója a decentralizáció és a zéró bizalom; a Bitcoinnal kapcsolatos műveletek elvégzéséhez nem szükséges központi szolgáltatás. Az alkalmazás jó választás fejlesztői háttérrel nem rendelkező felhasználók számára."
+    walletedgewallet: "Az Edge egy Segwit kompatibilis mobile bitcoin tárca ami magas szintű adatvédelmet, biztonságot és decentralizációt garantál, mely nagyon könnyen használható mindenki számára. Az Edge tárca minden esetben automatikusan titkosításra, biztonsági mentésre kerül és még akkor is működik amikor az Edge szerverek nem elérhetőek."
```

Comment:
Please revise or drop change to this file as per comment in: `_translations/en.yml`